docs/system_overview/federation.md

```diff
@@ -7,27 +7,81 @@ Primary reasons for using Zone Federation include:
 
 ## Setup
 
-To federate Zone A and Zone B, administrators in each zone must:
+To federate ZoneA and ZoneB, administrators in each zone must:
 
-1. Coordinate and share their `catalog_provider_hosts`, `negotiation_key`, `zone_key`, and `zone_name` information
-2. Define the remote zone in their respective iCAT, and
-3. Define any remote users in their respective iCAT before any access permissions can be granted.
+1. Coordinate and share their `catalog_provider_hosts`, `negotiation_key`, `zone_key`, `zone_name`, and `zone_port` information.
+2. Verify that the remote host name can be resolved.
+3. Define the remote zone in their respective iCAT.
+4. Define any remote users in their respective iCAT before any access permissions can be granted.
 
+### Example
 
-In Zone A, add Zone B and define a remote user:
+Let ZoneA have the following properties:
+~~~
+host name: zoneA-provider.example.org
+zone name: ZoneA
+zone key : ZONE_KEY_FOR_ALL_OF_ZONE_A
+zone port: 1247
+~~~
+
+and ZoneB have the following properties:
+~~~
+host name: zoneB-provider.example.org
+zone name: ZoneB
+zone key : ZONE_KEY_FOR_ALL_OF_ZONE_B
+zone port: 2247
+~~~
+
+To federate these zones, the following must be done:
+
+For each zone, add an object to the `federation` array in `server_config.json`. The object must contain the remote zone's information.
+
+#### ZoneA's server_config.json
+~~~
+"federation": [
+    {
+        "catalog_provider_hosts": ["zoneB-provider.example.org"], 
+        "negotiation_key": "_____32_byte_pre_shared_key_____",
+        "zone_key": "ZONE_KEY_FOR_ALL_OF_ZONE_B", 
+        "zone_name": "ZoneB", 
+        "zone_port": 2247
+    }
+]
+~~~
+
+#### ZoneB's server_config.json
+~~~
+"federation": [
+    {
+        "catalog_provider_hosts": ["zoneA-provider.example.org"], 
+        "negotiation_key": "_____32_byte_pre_shared_key_____",
+        "zone_key": "ZONE_KEY_FOR_ALL_OF_ZONE_A", 
+        "zone_name": "ZoneA", 
+        "zone_port": 1247
+    }
+]
+~~~
+
+For more information, see [Server Authentication](#server-authentication).
+
+It is important that the servers are able to resolve the host name of the federated zone. This can be handled in many ways. The resolution could come from this account's `hosts_config.json` file, this server's `/etc/hosts` file, or DNS in priority order respectively.
+
+With the server configuration and networking complete, all that is left is to define the remote zone and users in the catalog.
```

Comment:
awkward construction... "all that is left is to define"...

how about...

"the final step is to define"